test/EndToEnd.Tests/GivenWindowsApp.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using EndToEnd.Tests.Utilities;
+
+namespace EndToEnd.Tests
+{
+    public class GivenWindowsApp(ITestOutputHelper log) : SdkTest(log)
+    {
+        [WindowsOnlyTheory]
+        [InlineData("10.0.17763.0")]
+        [InlineData("10.0.18362.0")]
+        [InlineData("10.0.19041.0")]
+        [InlineData("10.0.20348.0")]
+        [InlineData("10.0.22000.0")]
+        [InlineData("10.0.22621.0")]
+        // Skipped due to: https://github.com/dotnet/sdk/pull/42090/files#r1680016439
+        //[InlineData("10.0.26100.0")]
```

Comment:
This was a recently added version. The test fails against it. I've logged this details [here](https://github.com/dotnet/sdk/pull/42090/files#r1680016439) and notified @manodasanW about it. For now, I'm skipping as it shouldn't block the entire PR. It can be addressed in a different PR to enable it.